tests/phpunit/tests/media.php

```diff
@@ -5283,11 +5283,30 @@ public function test_quality_with_image_conversion_file_sizes() {
 		// The main (scaled) image: the JPEG should be smaller than the WebP.
 		$this->assertLessThan( $webp_sizes['filesize'], $jpeg_sizes['filesize'], 'The JPEG should be smaller than the WebP.' );
 
-		// Sub-sizes: for each size, the JPEGs should be smaller than the WebP.
+		// Sub-sizes: for each size, the WebP should be smaller than the JPEG.
 		$sizes_to_compare = array_intersect_key( $jpeg_sizes['sizes'], $webp_sizes['sizes'] );
 		foreach ( $sizes_to_compare as $size => $size_data ) {
 			$this->assertLessThan( $webp_sizes['sizes'][ $size ]['filesize'], $jpeg_sizes['sizes'][ $size ]['filesize'] );
 		}
+
+		// Repeat the size tests with AVIF images.
+		add_filter( 'image_editor_output_format', array( $this, 'image_editor_output_avif' ) );
+		$avif_sizes = wp_generate_attachment_metadata( $attachment_id, $file );
+		remove_filter( 'image_editor_output_format', array( $this, 'image_editor_output_avif' ) );
+
+		// Sub-sizes: for each size, the AVIF should be smaller than the JPEG.
+		$sizes_to_compare = array_intersect_key( $jpeg_sizes['sizes'], $avif_sizes['sizes'] );
+		foreach ( $sizes_to_compare as $size => $size_data ) {
+			$this->assertLessThan( $avif_sizes['sizes'][ $size ]['filesize'], $jpeg_sizes['sizes'][ $size ]['filesize'] );
```

Comment:
Something goes wide with the assert. The reverse check is passed but why 🤔

Reply:
thanks for reviewing. I'll take a look and make sure high quality is used for the first pass.

Reply:
🤞🏼 fixed the unit tests in bf60efc7afab9e631db8324892d9599acc263905